Client.xcodeproj/project.pbxproj

```diff
@@ -3641,6 +3641,9 @@
 							com.apple.ApplicationGroups.iOS = {
 								enabled = 1;
 							};
+							com.apple.BackgroundModes = {
```

Comment:
Do we not also need to add `audio` to a `UIBackgroundModes` array in `Info.plist`?